internal/resources/storage.go

```diff
@@ -42,6 +43,38 @@ func (b *StorageClusterBuilder) GetGRPCEndpoint() string {
 	return fmt.Sprintf("%s:%d", host, api.GRPCPort)
 }
 
+func (b *StorageClusterBuilder) appendCAConfigMapIfNeeded(optionalBuilders []ResourceBuilder) []ResourceBuilder {
+	additionalCAs := make(map[string]string)
+
+	if b.Spec.CaBundle != "" {
+		data, _ := base64.StdEncoding.DecodeString(b.Spec.CaBundle)
+		additionalCAs["generalRoot.crt"] = string(data)
+	}
+
+	// TODO figure out how to get data from secret
```

Comment:
TODO: extract the certificate authority from grpc and interconnect services and put it to system trust store as well